### PR TITLE
Delay use case

### PR DESCRIPTION
**Pull request by arandaaron04**
## Description 
Calculates the Delayed time using Scheduled Time and Computed Time.

## Details
- Created Delay file that has 2 methods: delayTime and calculated
- delayTime used in StationInfoInteractor
- dealy left as null for error issues (no need to change)
- Will return a message based on delay in IncomingVehicles panel

## Relevant issues
- May interfere will future changes
- uses parsed, must work around or implement

### CODE REVIEW

File: src/use_case/delay/Delay.java
Comment (jacqnnn) [inline src/use_case/delay/Delay.java:1]: Might need to move it to somewhere else

**Review by carmen-chau — CHANGES_REQUESTED** [review]
Good work so far, please convert your code structure so that it adheres to clean architecture. Once you're done, please re-request a review

File: src/use_case/delay/Delay.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        String computedDate = computed.substring(11);
+
+        DateFormat dateFormat = new SimpleDateFormat("hh:mm:ss");
+        Date scheduleTime = dateFormat.parse(scheduleDate);
+        Date computedTime = dateFormat.parse(computedDate);
+
+        double difference = (computedTime.getTime() - scheduleTime.getTime()) * 0.001; // in seconds
+
+        // if cases for returning the differences in time
+        if (difference >= 3600) {
```
[the remark this answers is not in the document]
Comment (arandaaron04) [reply src/use_case/delay/Delay.java]: Message updated accordingly.

**Review by jacqnnn — CHANGES_REQUESTED** [review]
Probably need to do update and pull main due to main changes

**Pull request merged**